test/e2e/framework/kcp.go

```diff
@@ -388,7 +388,7 @@ func (c *kcpServer) defaultConfig() (*rest.Config, error) {
 func (c *kcpServer) DefaultConfig(t *testing.T) *rest.Config {
 	cfg, err := c.defaultConfig()
 	require.NoError(t, err)
-	return cfg
+	return rest.AddUserAgent(rest.CopyConfig(cfg), t.Name())
```

Comment:
This is a slightly unrelated change but very valuable.

Reply:
To my own knowledge, how is the helpful? for when looking at the logs of the requests to the server?

Reply:
Yep, if you look at API server logs with this change you can differentiate test clients.

Reply:
I like this.